static/app/data/platformCategories.tsx

```diff
@@ -378,20 +374,6 @@ export const releaseHealth: PlatformKey[] = [
   'native-crashpad',
   'native-breakpad',
   'native-qt',
-  'electron',
-  'javascript-electron',
-  'rust',
-  'php',
-  'php-laravel',
-  'php-symfony',
-  'dotnet',
-  'dotnet-awslambda',
-  'dotnet-gcpfunctions',
-  'dotnet-maui',
-  'dotnet-uwp',
-  'dotnet-wpf',
-  'dotnet-winforms',
-  'dotnet-xamarin',
```

Comment:
Why is this deleted, according to the excel sheet, this is automatically supported?